<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AP00084VDI" failures="1" tests="6" name="com.iceridge.testcases.CreateMatterTest" time="242.084" errors="0" timestamp="25 Sep 2019 04:17:17 GMT" skipped="0">
  <testcase classname="com.iceridge.testcases.CreateMatterTest" name="matterCreation" time="45.930"/>
  <testcase classname="com.iceridge.testcases.CreateMatterTest" name="matterCreation" time="45.829"/>
  <testcase classname="com.iceridge.testcases.CreateMatterTest" name="matterCreation" time="14.375">
    <failure message="Matter details are not added succesfully expected [Antonio] but found [Antonio Criola]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Matter details are not added succesfully expected [Antonio] but found [Antonio Criola]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.iceridge.testutilities.TestUtil.verifyDetails(TestUtil.java:210)
at com.iceridge.testcases.CreateMatterTest.matterCreation(CreateMatterTest.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- matterCreation -->
  <testcase classname="com.iceridge.testcases.CreateMatterTest" name="matterCreation" time="44.813"/>
  <testcase classname="com.iceridge.testcases.CreateMatterTest" name="matterCreation" time="46.170"/>
  <testcase classname="com.iceridge.testcases.CreateMatterTest" name="matterCreation" time="44.967"/>
</testsuite> <!-- com.iceridge.testcases.CreateMatterTest -->
